data/darktableconfig.xml.in

```diff
@@ -2084,6 +2084,13 @@
     <shortdescription>show scrollbars for central view</shortdescription>
     <longdescription>defines whether scrollbars should be displayed</longdescription>
   </dtconfig>
+  <dtconfig prefs="lighttable" section="general">
+     <name>lighttable/ui/milliseconds</name>
+     <type>bool</type>
+     <default>false</default>
+     <shortdescription>show image datetime milliseconds</shortdescription>
+     <longdescription>defines whether datetime milliseconds should be displayed</longdescription>
```

Comment:
`defines whether time should be displayed with milliseconds`

Reply:
done too!